tests/sqllogictests/suites/mode/standalone/explain/update.test

```diff
@@ -100,6 +115,25 @@ CommitSink
                 ├── push downs: [filters: [is_true(t1.b (#1) > 2)], limit: NONE]
                 └── estimated rows: 4.00
 
+query T
+explain analyze partial update t1 set a = a + 1 where a in (select a from t2) and b > 2;
```

Comment:
so the explain does not have MutationSource ?

Reply:
We need more tests to verify `explain/explain analyze update/delete` don't perform actual updates/deletes.

Reply:
> so the explain does not have MutationSource ?

`MutationSource` is only generated in update/delete statements without subqueries. The processing flow for update/delete statements with subqueries is similar to merge into.